src/client/app/components/ChartSelectComponent.jsx

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { ToggleButtonGroup, ToggleButton } from 'react-bootstrap';
+import { chartTypes } from '../reducers/graph';
+
+/**
+ * A component that allows users to select which chart should be displayed.
+ */
+export default class ChartSelectComponent extends React.Component {
```

Comment:
Probably cleaner written as a functional component. Just call `onChange={value => props.changeChartType(value)}` in your render method